main.go

```diff
@@ -47,14 +45,18 @@ func ProcessCertificate(in <-chan string, wg *sync.WaitGroup) {
 		}
 		parsed, err := x509.ParseCertificate(der)
 		if err != nil { //could not parse
-			//log.Info("Could not parse certificate with error:", err)
+			if crashIfParseFail {
+				log.Fatal("Could not parse certificate with error: ", err)
```

Comment:
Log statements and error messages should start lowercase, because they are often concatenated: `statement: substatement: that had it's own substatement`